runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/BeginnerMapClue.java

```diff
@@ -0,0 +1,113 @@
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.cluescrolls.clues;
+
+import com.google.common.collect.ImmutableSet;
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.util.Set;
+import lombok.Getter;
+import net.runelite.api.coords.LocalPoint;
+import net.runelite.api.coords.WorldPoint;
+import net.runelite.api.widgets.WidgetID;
+import net.runelite.client.plugins.cluescrolls.ClueScrollPlugin;
+import net.runelite.client.ui.overlay.OverlayUtil;
+import net.runelite.client.ui.overlay.components.LineComponent;
+import net.runelite.client.ui.overlay.components.PanelComponent;
+import net.runelite.client.ui.overlay.components.TitleComponent;
+
+@Getter
+public class BeginnerMapClue extends ClueScroll implements LocationClueScroll
+{
+	private static final Set<BeginnerMapClue> CLUES = ImmutableSet.of(
+		new BeginnerMapClue(WidgetID.BEGINNER_CLUE_MAP_CHAMPIONS_GUILD, new WorldPoint(3166, 3361, 0), "South West of the Champion's Guild"),
+		new BeginnerMapClue(WidgetID.BEGINNER_CLUE_MAP_VARROCK_EAST_MINE, new WorldPoint(3290, 3374, 0), "Outside Varrock East Mine"),
+		new BeginnerMapClue(WidgetID.BEGINNER_CLUE_MAP_DRAYNOR, new WorldPoint(3093, 3226, 0), "South of Draynor Village Bank"),
+		new BeginnerMapClue(WidgetID.BEGINNER_CLUE_MAP_NORTH_OF_FALADOR, new WorldPoint(3043, 3398, 0), "In the standing stones north of Falador"),
+		new BeginnerMapClue(WidgetID.BEGINNER_CLUE_MAP_WIZARDS_TOWER, new WorldPoint(3110, 3152, 0), "On the south side of the Wizard's Tower")
+	);
+
+	private final int widgetGroupID;
+	private final WorldPoint location;
+	private final String description;
+
+	private BeginnerMapClue(int widgetGroupID, WorldPoint location, String description)
+	{
+		this.widgetGroupID = widgetGroupID;
+		this.location = location;
+		this.description = description;
+		setRequiresSpade(true);
+	}
+
+	@Override
+	public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plugin)
+	{
+		panelComponent.getChildren().add(TitleComponent.builder().text("Map Clue").build());
+
+		panelComponent.getChildren().add(LineComponent.builder()
+			.left("Click the clue scroll along the edge of your world map to see your destination.")
+			.build());
+
+		panelComponent.getChildren().add(LineComponent.builder()
+			.left("Travel to the destination and dig on the marked tile.")
+			.build());
+
+		if (description != null)
+		{
+			panelComponent.getChildren().add(LineComponent.builder().build());
+			panelComponent.getChildren().add(LineComponent.builder()
+				.left(description)
+				.build());
+		}
+	}
+
+	@Override
+	public void makeWorldOverlayHint(Graphics2D graphics, ClueScrollPlugin plugin)
```

Comment:
You should just extend MapClue, or just use it, you literally copy-pasted the code